dev/io.openliberty.org.jboss.resteasy.common/src/io/openliberty/restfulWS/config/ConfigImpl.java

```diff
@@ -35,8 +35,20 @@
             });
         }
         for (Object key : System.getProperties().keySet()) {
+            if (key.equals("resteasy.unwrapped.exceptions")) {
+                
+            }
             configProperties.put((String) key, System.getProperty((String)key));
         }
+        // Add EJBExcption to the list of wrapped exceptions that are processed by RESTEasy
+        if (configProperties.containsKey("resteasy.unwrapped.exceptions")) {
+            String value = configProperties.get("resteasy.unwrapped.exceptions");
+            if (!(value.contains("jakarta.ejb.EJBException"))) {
+                configProperties.replace("resteasy.unwrapped.exceptions", value + ",jakarta.ejb.EJBException");               
+            }
+        } else {
+            configProperties.put("resteasy.unwrapped.exceptions", "jakarta.ejb.EJBException");       
+        }
```

Comment:
You may be able to simplify this with:
```suggestion
        configProperties.merge("resteasy.unwrapped.exceptions", "jakarta.ejb.EJBException", (oldVal, newVal) -> oldVal + "," + newVal);
            
```

Reply:
Thanks.   I'm assuming I'll need this both inside the doPriv bock and outside?

Reply:
no, adding to the list does not require Java 2 security.  The doPriv only needs to go around the code that gets system properties.  so the code to merge can go after the system properties are merged into the hashmap

Reply:
Switched to 
        configProperties.merge("resteasy.unwrapped.exceptions", "jakarta.ejb.EJBException", (oldVal, newVal) -> {
            return (oldVal.contains("jakarta.ejb.EJBException") ? oldVal : oldVal + "," + newVal); });
Done